rcsbapi/const.py

```diff
@@ -37,11 +32,40 @@ class Const:
     RCSB_SEARCH_API_QUERY_URL: str = "https://search.rcsb.org/rcsbsearch/v2/query"
     UPLOAD_URL: str = "https://user-upload.rcsb.org/v1/putMultipart"
     RETURN_UP_URL: str = "https://user-upload.rcsb.org/v1/download/"
-    #
+
+    SEARCH_API_SCHEMA_DIR = "search/resources"
+    SEARCH_API_SCHEMA_BASE_URL = "http://search.rcsb.org/rcsbsearch/v2/metadata/"
+    STRUCTURE_SCHEMA_FILE_NAME = "structure_schema.json"
+    CHEMICAL_SCHEMA_FILE_NAME = "chemical_schema.json"
+    SEARCH_API_SCHEMA_FILE_TO_ENDPOINT = MappingProxyType({
+        STRUCTURE_SCHEMA_FILE_NAME: "schema",
+        CHEMICAL_SCHEMA_FILE_NAME: "chemical/schema",
+    })
+    STRUCTURE_ATTRIBUTE_SCHEMA_URL: str = SEARCH_API_SCHEMA_BASE_URL + SEARCH_API_SCHEMA_FILE_TO_ENDPOINT[STRUCTURE_SCHEMA_FILE_NAME]
+    STRUCTURE_ATTRIBUTE_SCHEMA_FILE: str = SEARCH_API_SCHEMA_DIR + "/" + STRUCTURE_SCHEMA_FILE_NAME
+    CHEMICAL_ATTRIBUTE_SCHEMA_URL: str = SEARCH_API_SCHEMA_BASE_URL + SEARCH_API_SCHEMA_FILE_TO_ENDPOINT[CHEMICAL_SCHEMA_FILE_NAME]
+    CHEMICAL_ATTRIBUTE_SCHEMA_FILE: str = SEARCH_API_SCHEMA_DIR + "/" + CHEMICAL_SCHEMA_FILE_NAME
```

Comment:
If you want the structure and chemical attribute schemas to have their own dedicated constant (rather than using a dictionary), then I would suggest doing the following:

```suggestion
    SEARCH_API_SCHEMA_DIR: str = "search/resources"
    SEARCH_API_STRUCTURE_ATTRIBUTE_SCHEMA_URL: str = "http://search.rcsb.org/rcsbsearch/v2/metadata/schema"
    SEARCH_API_STRUCTURE_ATTRIBUTE_SCHEMA_FILENAME: str = "structure_schema.json"
    SEARCH_API_CHEMICAL_ATTRIBUTE_SCHEMA_URL: str = "https://search.rcsb.org/rcsbsearch/v2/metadata/chemical/schema"
    SEARCH_API_CHEMICAL_ATTRIBUTE_SCHEMA_FILENAME: str = "chemical_schema.json"
```

Then, in the adjacent code that saves and/or loads the local schema files (e.g., in `test_search_schema.py`), you should join the `SEARCH_API_SCHEMA_DIR` and `...FILENAME` there, not here. E.g.:
```python
os.path.join(const.SEARCH_API_SCHEMA_DIR, const.SEARCH_API_STRUCTURE_ATTRIBUTE_SCHEMA_FILENAME)
```

Reply:
Thanks, I like this better